src/config/mod.rs

```diff
@@ -111,11 +111,11 @@ macro_rules! set_from_flag {
 
 impl Config {
 
-    pub fn default() -> Config {
+    pub fn play_out_aftermath(play_out_aftermath: bool) -> Config {
```

Comment:
You can't just modify `Arc<Config>` directly, you need to initialize it to the proper value, otherwise it would be something like `Arc<RefCell<Config>>` which is completely unnecessary because, again, we only "change" it once